mate-panel/panel-run-dialog.c

```diff
@@ -744,6 +763,23 @@ panel_run_dialog_find_command_idle (PanelRunDialog *dialog)
 					    -1);
 		}
 
+		gtk_tree_model_get (model, &iter, COLUMN_VISIBLE, &visible, -1);
+		if (visible && visible_program_idx < G_N_ELEMENTS (accelerator_key_mapping)) {
+			gtk_list_store_set (dialog->program_list_store,
+								&iter,
+								COLUMN_ACCELERATOR_MASK, (gint)accelerator_key_mapping[visible_program_idx].modifier,
+								COLUMN_ACCELERATOR_KEY_VALUE, accelerator_key_mapping[visible_program_idx].key_id,
+								-1);
```

Comment:
Formatting issue. Too much tabs are used in this block. Tab size seems 8 in this file.

Reply:
All formatting issues were fixed using https://gist.github.com/mattarroz/6d32243ece7445efc7e71ea6e6202b4a

Maybe we should consider adding a .clang-format file to the source tree?

(fixup 6cff6f3d2e9fd695399328e5597ac7bfbad2e2ab)

Reply:
How does this work?